examples/canon.js

```diff
@@ -1,10 +1,9 @@
-/**
-  Simple test middleware function
-*/
-function exampleReduxMiddleware(canonMiddleware) {
-  return canonMiddleware;
-}
+const {modelPaths: coreModelPaths} = require("@datawheel/canon-core/models");
+const {modelPaths: cmsModelPaths} = require("@datawheel/canon-cms/models");
```

Comment:
Can you add these dependencies to the installation steps for the CMS (...and eventually the logins module)?